examples/byoc-with-sagemaker-containers/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+IMAGE_NAME=scikit-learn-image
```

Comment:
Maybe it is just me, but I feel this script... doesn't do much?

Most of the other example notebooks that have a build script, end up tagging and pushing to ECR as well. Which seems to imply this is mean't for local mode.